tests/e2e/specs/SignalEndEvent.spec.js

```diff
@@ -0,0 +1,15 @@
+import { addNodeTypeToPaper, assertDownloadedXmlContainsExpected } from '../support/utils';
+import { nodeTypes } from '../support/constants';
+
+describe('Signal Start Event', () => {
+  it('Can create signal start event', () => {
```

Comment:
Missed this: Should re-name "start" to "end".

Reply:
Uh, yep